### PR TITLE
Geolift - first docs

### PR DESCRIPTION
**Pull request by gregdale**
Opening PR for Geolift docs

### CODE REVIEW

File: docs/quick-starts/geolift-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 3
+---
```
Comment (tbuffington7) [inline docs/quick-starts/geolift-quickstart.md:3]:
```suggestion
---
sidebar_position: 3
---
```

---
slug: /geolift-quickstart
sidebar_position: 4
---

File: docs/geolift/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+3. **Need historical data:** Although Eppo supports methods like CUPED for traditional experiments that use historical data to reduce variance and increase power, it does not require it. In Geolift, historical data is needed to develop the synthetic control.
+4. **Stronger statistical assumptions:** Since we can't rely on randomization to accomodate any differences between the treatment and control groups, there is a stricter set of assumptions users should follow to ensure the units are similar enough to be compared in the quasi-experiment. For more detail, see [Assumptions and Best Practices](/geolift/assumptions_best_practices/).
+
+## Historical Data Needed
+
+Typically, we use your existing Eppo metrics and so will use up to 18 months of the metric as available in your data warehouse. More the better, but on the low end, if you are setting up a new data source and metric, we recommend at least three months of historical data before designing a test.
```
Comment (emetelka) [inline docs/geolift/index.md:36]:
```suggestion
Geolift tests use the metrics you create in Eppo, which are based on your data warehouse. We recommend at least three months of historical data before designing a test and recommend up to 18 months of data if available.
```

File: docs/quick-starts/geolift-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+slug: /geolift-quickstart
+sidebar_position: 8
+---
+# Creating your first Geolift test
+
+This guide will walk you through creating your first Eppo Geolift quasi-experiment. We will create a new quasi-experiment, specify the business question, specify test parameters, and submit the test to the power analysis and simulation pipeline.
+
```
Comment (emetelka) [inline docs/quick-starts/geolift-quickstart.md:8]: Can link to "creating your first metric" quickstart here

File: docs/quick-starts/geolift-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+2. **Create a Fact SQL for the events to be measured**: If you want to test the intervention’s effect on revenue by Country, you’ll want to create a Fact SQL that brings net revenue into Eppo along with the Country that the revenue took place in. More information on Facts here.
+3. **Create a Metric for the event to be measured**: Next, create a metric that aggregates the revenue by country created in the previous step into the metric that you’d like to examine in this test. When creating a new metric, select the Entity you just created, then the Fact and the Aggregation type desired. More information on Metrics here. Eppo Geolift currently supports SUM and COUNT metrics with no timeframes specified.
+
+## Test objective and data
+
+1. **Create a Quasi-experiment**: Now, you can specify the parameters for the Geolift test.
```
Comment (emetelka) [inline docs/quick-starts/geolift-quickstart.md:15]: Describe where to create a quasi experiment (In the analysis tab, click the create button and select quasi-experiment)

File: docs/quick-starts/geolift-quickstart.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+   3. **Idiosyncratic and unrepresentative behavior in a synthetic control region.** For example, if you sell sports merchandise or tickets and plan to run a nationwide advertising test during a major league championship, you may want to exclude the markets belonging to the teams playing in the series, as their behavior will be fundamentally and temporarily different from other markets. For more on assessing these dynamics, see [Statistical Assumptions and Best Practices](../geolift/assumptions_best_practices.md)
+
+![Modify eligible units](/img/geolift/geolift_create_page2.png)
+
+Once you’ve submitted the test objectives and parameters, the test is submitted into our planning pipeline. At this stage, Eppo will use the information provided to propose test and control groups, a 2-cell or 3-cell design, and a minimum detectable effect estimation.
+
+Once a test is launched, the [Results Analysis](../geolift/analysis.md) screen will be enabled and visible from the Quasi-experiments tab of the Analysis section.
```
Comment (emetelka) [inline docs/quick-starts/geolift-quickstart.md:35]: Add a congrats message to the end, for example: Congratulations, you have now created your first quasi-experiment! If you want to analyze additional experiments, simply repeat this process with a different scenario

File: docs/geolift/analysis.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+Below, we include three plots per variant of the test:
+
+- **Absolute values**: The Metric compared to its synthetic control counterfactual as absolute values
+- **Difference in values**: The difference in values between the metric and its counterfactual
+- **Cumulative difference**: The cumulative sum of the difference in values
+
+![Geolift Results Analysis](/img/geolift/geolift_overview_tab.png)
+
+## Test Plan
```
Comment (emetelka) [inline docs/geolift/analysis.md:32]: Remove this

**Pull request merged**